### PR TITLE
Real Time Token Balances with the Balances REST API 

### LINKED ISSUE #4776: Use real-time token balance in balances API
### Problem

As a REST API user, I'd like to see my up to date token balance information when querying `/api/v1/balances` API.

### Solution

* Change `/api/v1/balances` to query `token_account.balance` when no `timestamp` parameter is provided

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by edwin-greene**
**Description**:
The Balances API will now use the token account table to obtain real time token balances when no timestamp parameter is present. 

**Related issue(s)**:

Fixes #4776 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — DISMISSED** [review]

File: hedera-mirror-rest/balances.js [hunk 1]
```diff
@@ -166,6 +167,18 @@ const getTokenBalanceSubQuery = (order, timestampColumn) => {
...
 
+const getTokenAccountBalanceSubQuery = (order) => {
+  return `
+    select json_agg(json_build_object('token_id', token_id, 'balance', balance))
+    from (
+      select token_id, balance
+      from token_account ta
+      where ta.account_id = ab.account_id
+      order by token_id ${order}
+      limit ${tokenBalanceLimit.multipleAccounts}
```
Comment (xin-hedera) [inline hedera-mirror-rest/balances.js:178]: should not show balance for those dissociated token account associations. this is to align with the token balance info from the account balance file. if an account dissociates itself from a token, in the next account balance file, the <account, token> entry will disappear.

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/balances.js (hunk 1 above)
Comment (edwin-greene) [reply hedera-mirror-rest/balances.js]: Great input. I have updated the query to only get associated token accounts.

File: hedera-mirror-rest/__tests__/specs/balances/no-params.json [hunk 2]
```diff
@@ -152,6 +152,39 @@
         "hash": "c73067d7e07eaab5a6156237f928acab0a9a2783083651996237a885230a2e0f147b322fb5d0e19a0c44cf3840bd3ba9",
         "name": "1970-01-01T00_00_10.000000001Z.rcd"
       }
+    ],
+    "tokenaccounts": [
+      {
+        "token_id": "0.0.90000",
+        "account_id": "0.15.01",
```
Comment (xin-hedera) [inline hedera-mirror-rest/__tests__/specs/balances/no-params.json:159]: nit, here and every other place the account is in the form of `x.y.0z`

```suggestion
        "account_id": "0.15.1",
```

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**